tapdb/universe.go

```diff
@@ -523,7 +523,7 @@ func universeFetchIssuanceProof(ctx context.Context,
 	}
 
 	if len(universeLeaves) == 0 {
-		return nil, ErrNoUniverseProofFound
+		return nil, nil
```

Comment:
Shouldn't we do this on another level? Sounds to me like we'd want to know if the DB doesn't have a certain proof. But maybe we don't want to return an error on the RPC interface?

Reply:
I've reverted my change such that `FetchIssuanceProof` returns `ErrNoUniverseProofFound` like it did before. I'll check the RPC side.